src/settings/MappingProfiles/MappingProfilesForm.js

```diff
@@ -162,6 +162,22 @@ export const MappingProfilesFormComponent = ({
       onLink: setAddedRelations,
       onUnlink: setDeletedRelations,
     },
+    'item-status': {
+      acceptedValuesList: [
+        { name: 'Available' },
+        { name: 'Awaiting pickup' },
+        { name: 'Awaiting delivery' },
+        { name: 'Checked out' },
+        { name: 'In process' },
+        { name: 'In transit' },
+        { name: 'Missing' },
+        { name: 'On order' },
+        { name: 'Paged' },
+        { name: 'Declared lost' },
+        { name: 'Order closed' },
+        { name: 'Claimed returned' },
+      ],
+    },
```

Comment:
Maybe is it better to move these data somewhere to constants?

Reply:
Done